server.R

```diff
@@ -0,0 +1,342 @@
+library(shiny)
+library(shinyalert)
+library(magrittr)
+library(readr)
+library(stringr)
+library(httr)
+library(dplyr)
+
+engine <- handle("http://18.132.35.89:8001/")
```

Comment:
Might want to keep the /v2/ in the handle, rather than at some point start making create calls at `/v3/` but join calls at `/v2/`.